resources/kernel/make_kernel.sh

```diff
@@ -111,10 +111,19 @@ make_initramfs() {
 # Exits with error if any other format is specified.
 validate_kernel_format() {
```

Comment:
The description above should be updated.

Reply:
Indeed, updated it.

Reply:
> Can you please make the commit message a bit clearer? I think the arm support involves more changes, e.g. #124.

Indeed arm support means more in the vmm-reference context. For Firecracker we need just these changes for the moment. I updated the commit message.